bin/commands/internal/start/prepare/index.ts

```diff
@@ -459,7 +458,7 @@ export function execute() {
     // other extensions need to specify `require_java` in their validate.sh
     java.requireJava();
   }
-  if (stringlib.itemInList('app-server', std.getenv('ZWE_CLI_PARAMETER_COMPONENT'))) {
+  if (stringlib.itemInList(std.getenv('ZWE_PRIVATE_CORE_COMPONENTS_REQUIRE_NODE'), std.getenv('ZWE_CLI_PARAMETER_COMPONENT'))) {
```

Comment:
This will work only because there is just `app-server` defined. If the list of the components is extended, `itemInList` uses the first parameter to search, but the second parameter is split by `,`.